api/v3/Contribution.php

```diff
@@ -616,84 +616,76 @@ function _civicrm_api3_contribution_completetransaction_spec(&$params) {
  * @throws API_Exception
  */
 function civicrm_api3_contribution_repeattransaction($params) {
+
   civicrm_api3_verify_one_mandatory($params, NULL, ['contribution_recur_id', 'original_contribution_id']);
+
+  // We need a contribution to copy.
   if (empty($params['original_contribution_id'])) {
```

Comment:
if it's empty we should do nothing in this layer - `BAO_Contribution::repeattransaction` is doing all tis stuff already